bigcache.go

```diff
@@ -137,6 +137,34 @@ func (c *BigCache) Get(key string) ([]byte, error) {
 	return shard.get(key, hashedKey)
 }
 
+// GetMulti reads entry for each of the keys.
+// It returns an ErrEntryNotFound when
+// no entry exists for the given key.
+func (c *BigCache) GetMulti(keys []string) ([][]byte, error) {
+
+    entries := make([][]byte,len(keys))
+    var err error
+    shardsLocked := make(map[uint64]bool)
```

Comment:
Prealloc

Reply:
the comment is vague :slightly_smiling_face: 

do you mean like?
```go
entries := make([][]byte, 0, len(keys))
```
and then use append?

Reply:
Yes, exactly that.